Core/Core/Api/GraphQL/Client.cs

```diff
@@ -116,42 +100,34 @@ internal async Task<T> ExecuteWithResiliencePolicies<T>(Func<Task<T>> func)
 
     var delay = Backoff.DecorrelatedJitterBackoffV2(TimeSpan.FromSeconds(1), 5);
     var graphqlRetry = Policy
-      .Handle<SpeckleGraphQLInternalErrorException<T>>()
+      .Handle<SpeckleGraphQLInternalErrorException>()
       .WaitAndRetryAsync(
         delay,
-        (ex, timeout, context) =>
+        (ex, timeout, _) =>
         {
-          var graphqlEx = (SpeckleGraphQLException<T>)ex;
-          SpeckleLog.Logger
-            .ForContext("graphqlExtensions", graphqlEx.Extensions)
-            .ForContext("graphqlErrorMessages", graphqlEx.ErrorMessages)
-            .Warning(
-              ex,
-              "The previous attempt at executing function to get {resultType} failed with {exceptionMessage}. Retrying after {timeout}.",
-              typeof(T).Name,
-              ex.Message,
-              timeout
-            );
+          SpeckleLog.Logger.Debug(
```

Comment:
I'm suspicions about this function, not because of the logging, but because this isn't the only retry policy we have, and I suspect that this one is not actually the one we are relying on...
This one retries on `SpeckleGraphQLInternalErrorException`, which I'm fairly confident that we shoulden't be retrying on at all. 
I've already had a chat about this with Fabians and he agrees, but would want to run this past Gergo before touching anything.

Reply:
Have the chat and raise a ticket if we should change it